en-US/Conductors_Companion/04_Running_Game_Sessions/Running_your_Game/Running_your_Game.md

```diff
@@ -1,7 +1,34 @@
 ## Running your Game
 
 > **Warning**
-> This section has not yet been transferred from the Google Doc.
+> We probably want some kind of introduction to the topic here.
+
+- Relax
+- Focus on your strong start
+- Listen to the players, and build off of the ideas they bring you
+- Trust your preparation to help you run a creative, flexible game
+- Ask the players to summarize the events of the previous game session
+- Draw players into the story by asking them to describe killing blows, define monster characteristics, and describe interesting events during travel
+- Imagine the world as a living place when building scenes and situations
+- Let the world and the NPCs react to the characters' actions
+- Use a mixture of combat styles, including theater of the mind, gridded maps, and abstract maps
+- Maintain a good pace by staying close to the action
+- Cycle between action and relaxation, and alternate upward and downward emotional beats
+- Use specific hopeful or fearful beats to send the action in a specific direction
+
+<!-- style:sidebar -->
+> **Virtual Tabletops (VTTs)** \
+> A VTT is an online play space that provides a shared table for remote groups.
+> Common features include maps, tokens, handouts, dice rollers, online books, and character sheet management.
+> Many of them automate and record encounter mechanics. Some groups use them in person to take advantage of these features.
+>
+> You don’t need a VTT to play online—some groups just use voice, video, or text chat, sometimes with shared whiteboard, documents, or dice rollers.
+> Choose whatever gives your players a clear, accessible, shared game experience.
+<!-- blank line -->
```

Comment:
This is needed, otherwise our linter will complain about a possibly unintended blank line in a quotation block. Adding this comment makes it clear, that these are supposed to be two separate quotation blocks.